doc/intro.md

```diff
@@ -58,38 +60,45 @@ where:
 
 ### Trend-based Cohort Component Model
 
-This method involves projecting births, deaths and migrations by age and sex.
+This method involves projecting births, deaths and migrations by age and sex and using that information to infer an overall population projection.
 
 See below the cohort component method we aim to provide:
 
 ![CCM](images/MVP_generic_CCM_flowchart.png)
 
-Note:
-
-We will ultimately provide four alternatives for projecting the components of change (i.e. fertiliy, mortality and migration): projected values or rates can either be fixed, or determined by applying a national trend.
-See below:
+### Fertility
 
-![CCM - projection alternatives](images/Component_projection_alternatives.png)
+The standard methodology for birth projections relies on applying age-specific fertility rates (ASFR) to the female population and splitting the estimated births into male and female using a standard ratio.
 
-For estimating the value or rate for the first year of projection (first projection year or base year), we will provide three alternatives. See below:
+### Mortality
 
-![CCM - base year alternatives](images/First_projection_year_projection_alternatives.png)
+Death projections are calculated by applying a set of age and sex specific mortality rates (ASMR), as is the standard methodology. A single rate for deaths per sex and age group is applied across all years
+projected.
 
+### Migration
 
+The net migration is calulated based on historic movement of both domestic and international migrants. Migrants are further characterised as either in- or out-migrants, combining this information to project the 
+net-migration.
 
-1) Fertility
+### Core
+The core module takes historic population data as well as projections from each of the components of change (above) to calculate the  population for each projected year.
 
-The standard methodology for birth projections relies on applying age-specific fertility rates (ASFR)
-to the female population and splitting the estimated births into male and female using a standard ratio.
+For estimating the value or rate for the first year of projection (first projection year or base year), we will provide three alternatives. See below:
 
+![CCM - base year alternatives](images/First_projection_year_projection_alternatives.png)
 
+Note:
 
-2) Mortality
+We will ultimately provide four alternatives for projecting the components of change (i.e. fertiliy, mortality and migration): projected values or rates can either be fixed, or determined by applying a national trend.
+See below:
 
```

Comment:
change to 'and 4 methods associated with calculating projections in further projection years.'